### DISCUSSION TITLE
Offline record in m3u-list

### DISCUSSION THREAD

**Discussion by zdens** [iptv-org/community#280] [Q&A] [answered]
Hi all,
I'm new in almost all IPTV questions.

I'm trying to configure Kodi Media Center and wondering what m3u record's `[Offline]` label is affecting on.

Specifically, [this index m3u](https://github.com/iptv-org/iptv/blob/master/channels/ru.m3u) contains the record 
> #EXTINF:-1 tvg-id="OTR.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="https://iptvx.one/icn/otr_ru.png" group-title="General",ОТР (720p) [Offline]
https://strm.yandex.ru/kal/otr/otr0.m3u8

and after I configure Kodi to use one of `https://iptv-org.github.io/iptv/*` m3u-s files as channels source there is no "ОТР" channel in any group. and there is no "ОТР" record in these m3u-lists. Fine.

So, am I correct that `Offline` label prevents such a record to be included in m3u-list on github.io?
If "yes", how was this record labeled as offline? if I will open the list with content from https://github.com/iptv-org/iptv/blob/master/channels/ru.m3u in VLC, for example, I do can find "ОТР" channel and it is not "offline": I can watch this channel, it works.

so, is there any way to get this channel in https://iptv-org.github.io/iptv/*` m3u-s?

**Comment by BellezaEmporium** [comment]
@Sphinxroot any ideas ?

**Comment by Dum4G** [comment, marked as answer]
Github Actions literally offer it's VMs from somwhere in US and so it can't ping a Geo-blocked stream. Looks like yandex as a source should be marked as Geo-blocked in it's entirety to avoid an incident like this

**Comment by Sphinxroot** [comment]
depend what he use in kodi  ... pvr simple client  ?  or other addon ... in playlistloader  they normaly all showed up whit tag   but playlistloader dont handle  #EXTVLCOPT: format

**Reply by zdens** [reply]
yes, I use pvr simple client in Kodi.

**Reply by zdens** [reply]
so, will "yandex as a source" be marked as Geo-blocked?

of course, I can manually download index-list for my region and  use it in Kodi but automatic update is much more preferable solution.

**Reply by Dum4G** [reply]
I'm planning to decouple yandex as a source and that's gonna take plenty of time since many channels are truly no longer broadcasted, some are under a paywall now and others has never been added. Stay tuned
